packages/shared/src/components/post/PostActions.tsx

```diff
@@ -99,8 +98,20 @@ export function PostActions({
     }
 
     if (post.downvoted) {
+      trackEvent(
+        postAnalyticsEvent(AnalyticsEvent.RemovePostDownvote, post, {
+          extra: { origin },
```

Comment:
Just to verify again, this is origin that comes from a component right?

Reply:
Same as https://github.com/dailydotdev/apps/pull/1899#discussion_r1242166049